### ISSUE TITLE
Live tiles

### ISSUE THREAD

**Issue by RustyRaptor** [Zren/plasma-applet-tiledmenu#17]
Adding a feature similar to the live update tiles in windows would be nice. For example a tile that fetches and displays data from github.

**Comment by Zren** [comment]
Like you want an embedded widget that shows... issues from a github repo? Is there an example Windows Live Tile?

I could possible create a "live tile" if someone dropped the url `https://github.com/Zren/plasma-applet-tiledmenu` or `https://github.com/Zren/plasma-applet-tiledmenu/issues` onto the menu. I'd need to package everything in the "tiled menu" widget though. It's not really ideal since I'd need to develop every "Live Tile" myself rather than using existing widgets.

To do this properly, i'd need to make my widget a "Containment" like the system tray or the panel, which can "embed" other widgets inside themselves.

**Comment by RustyRaptor** [comment]
Sorry for not clarifying the github part. My understanding is it's really just an RSS feed so I think just implementing an "RSS Tile" would do the trick for a lot of things. I think it would also work for a weather tile as well depending on the API. 

https://github.com/Difegue/SimpleRSSLiveTile

**Comment by Zren** [comment]
A user mentioned the possible concept of an embedded WebView from a locale html file. This would probably be heavy on resources though.

![](
![2018-11-15___13-39-29](https://user-images.githubusercontent.com/416367/48574041-1eae7900-e8dc-11e8-9531-400f437fceb0.png)

**Comment by RustyRaptor** [comment]
It most likely would be heavy on resources but would be cool to have the option.

**Comment by Vistaus** [comment]
Yeah, this would be cool to have for sure :) Even it'd be heavy on resources, but maybe a warning could be added when creating a Live Tile ("WARNING: this will likely be heavy on resources" or something)?

**Comment by FireFragment** [comment]
*Live tiles* could also be another KDE widgets displayed inside the menu.
